lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -1294,6 +1302,29 @@ static bool DeserializeAllCompilerFlags(swift::CompilerInvocation &invocation,
                 << getImportFailureString(result) << "\n";
           return false;
         }
+
+        // In case of explicit modules, for now fallback to implicit
+        // module loading.
+        // TODO: Incorporate loading explicit module dependencies to
+        // speedup dependency resolution.
+        if (llvm::find(invocation.getExtraClangArgs(), "-fno-implicit-modules")) {
+          auto matchExplicitBuildOption = [](StringRef &arg) {
+            for (const auto &option : knownExplicitModulePrefixes)
+              if (arg.consume_front(option))
+                return true;
```

Comment:
It seems this doesn't need to be mutating the arguments, but maybe I've missed something?

```suggestion
              if (arg.starts_with(option))
                return true;
```

Reply:
Changed to `starts_with`, thanks!